CMakeLists.txt

```diff
@@ -205,6 +206,14 @@ if (CMAKE_SYSTEM_NAME STREQUAL "Windows")
     configure_file( ${CMAKE_CURRENT_SOURCE_DIR}/properties.h.in ${CMAKE_CURRENT_SOURCE_DIR}/properties.h @ONLY)
 endif()
 
+FetchContent_Declare(
+  nlohmann_json
+  GIT_REPOSITORY https://github.com/nlohmann/json.git
+  GIT_TAG origin/develop
```

Comment:
do we should wait until 1.12.1 are release ?

Reply:
This is because nlohmann json has a bug. If I do this, I can't develop the feature.

Reply:
it's exactly my question of should we merge with this but if you don't think so then it's good